libvast/src/chunk.cpp

```diff
@@ -192,7 +192,7 @@ chunk::chunk(view_type view, deleter_type&& deleter) noexcept
   : view_{view}, deleter_{std::move(deleter)} {
   auto data = view.data();
   auto sz = view.size();
-  VAST_TRACEPOINT(chunk_make, data, sz);
+  VAST_TRACE_SCOPEPOINT(chunk_make, data, sz);
```

Comment:
That does not look correct, your replacement sed should've included the opening bracket.

Reply:
Whoops, updated.